packages/apollo-modules-node/index.js

```diff
@@ -84,7 +84,7 @@ const _addTypes = (master, donor) => {
     ...master,
     schemaTypes: mergedSchemaTypes,
     typeDefs: [...master.schema, ...mergedSchemaTypes],
-    resolvers: _.merge(donor.typeResolvers, master.resolvers)
+    resolvers: _.merge(master.resolvers, donor.typeResolvers)
```

Comment:
This seems to cause `packages/auth/graphql/resolvers/User.js#email` to win over `servers/republik/graphql/resolvers/User.js#email`. Preventing expected email field exposure.

Test case: try to get an email via an custom pledge access token:
```json
{
  "data": {
    "signedIn": null,
    "me": {
      "email": null,
      "name": "X",
      "customPackages": [
        {
          "id": "Y"
        }
      ]
    }
  }
}
```
the same query bevor this changes does return an email (stg vs prd right now)

Reply:
good catch! In the schema declaration of our servers we expect the local module to win, thats why we actually need donor <- master here, which is a little bit confusing, but I don't have a better idea now.
I now clone `donor.typeResolvers` to resolve the actual problem of it being modified in the process.